<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>884</width>
    <height>631</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QPushButton {


width:30;
height:20;
  padding: 0.2em 0.2em 0.3em 0.2em;
    border: 1px solid rgb(100, 100, 100);
    background: rgb(170, 170, 127);
    

}
QPushButton:hover {
    background-color:rgb(240, 228, 230);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }

</string>
  </property>
  <property name="iconSize">
   <size>
    <width>2000</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="save">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
                    text-align: center;
               
    border-radius: 20px;


}
QPushButton:hover {
    background-color:rgb(240, 228, 230);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
                    text-align: center;
                border-radius: 20px;
               


}

QPushButton:hover {
    background-color:rgb(240, 228, 230);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="entity">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>40</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_entity">
    <property name="geometry">
     <rect>
      <x>780</x>
      <y>40</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
                    text-align: center;
                border-radius: 20px;
            


}
QPushButton:hover {
    background-color:rgb(240, 228, 230);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
    </property>
    <property name="text">
     <string>Add Entity</string>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QTableWidget" name="entity_view">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>211</width>
      <height>481</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="autoScrollMargin">
     <number>7</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QTableWidget" name="result_view">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>231</width>
      <height>321</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoScrollMargin">
     <number>7</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
   </widget>
   <widget class="QTextEdit" name="edit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>521</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
   </widget>
   <widget class="QTextEdit" name="result">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>240</y>
      <width>281</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="Next">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>30</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
                    text-align: center;
               
    border-radius: 20px;


}
QPushButton:hover {
    background-color:rgb(240, 228, 230);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
    </property>
    <property name="text">
     <string>Next</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../pencil.png</normaloff>../pencil.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>30</width>
      <height>30</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="txt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>470</y>
      <width>131</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
                    text-align: center;
               
    border-radius: 20px;


}
QPushButton:hover {
    background-color:rgb(240, 228, 230);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
    </property>
    <property name="text">
     <string>save in txt form</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>470</y>
      <width>121</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
 
                    text-align: center;
               
    border-radius: 20px;


}
QPushButton:hover {
    background-color:rgb(240, 228, 230);
   
}
QPushButton:pressed {
    background-color: rgb(217, 217, 217); }</string>
    </property>
    <property name="text">
     <string>clear</string>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>884</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
